build.zig

```diff
@@ -1,11 +1,19 @@
 const std = @import("std");
 
+const OS_TAG = @import("builtin").os.tag;
```

Comment:
Style nit: I'd rather use `const builtin = @import("builtin");` and use `builtin.os.tag == .windows` in the `if` statements. This is more idiomatic I think.